dot/rpc/modules/dev.go

```diff
@@ -69,3 +73,23 @@ func (m *DevModule) Control(r *http.Request, req *[]string, res *string) error {
 	}
 	return err
 }
+
+// SlotDuration Dev RPC to return slot duration
+func (m *DevModule) SlotDuration(r *http.Request, req *EmptyRequest, res *string) error {
+	var err error
```

Comment:
Why do we return error?

Reply:
I believe all JSON-RPC funcs need to return error? correct me if I'm wrong @edwardmack

Reply:
Yes, that's correct.